flow/designs/asap7/sha3/config.mk

```diff
@@ -0,0 +1,20 @@
+export PLATFORM               = asap7
+export CORNER                ?= BC
```

Comment:
Do we need this?

Reply:
@maliberty as platform reference we keep this variable. For ASAP7 all libraries defined specific to CORNER. so without this right libraries can't be chosen.